jin/1주차 과제/6.7.py

```diff
@@ -0,0 +1,2 @@
+a, b, c, d = map(int, input().split())
+print((a+b+c+d)//4)
```

Comment:
```suggestion
print(int((a+b+c+d)//4))
```
이런 방법으로 출력하는 것도 괜찮을 것 같아요~

Reply:
우와! 감사합니다! ㅎㅎ